modules/cluster-logging-deploy-cli.adoc

```diff
@@ -308,35 +315,42 @@ spec:
   managementState: "Managed"  <2>
   logStore:
     type: "elasticsearch"  <3>
+    retentionPolicy: <4>
+      application:
+        maxAge: 1d
+      infra:
+        maxAge: 7d
+      audit:
+        maxAge: 7d
     elasticsearch:
-      nodeCount: 3 <4>
+      nodeCount: 3 <5>
       storage:
-        storageClassName: "<storage-class-name>" <5>
+        storageClassName: "<storage-class-name>" <6>
         size: 200G
       redundancyPolicy: "SingleRedundancy"
   visualization:
-    type: "kibana"  <6>
+    type: "kibana"  <7>
     kibana:
       replicas: 1
   curation:
-    type: "curator"  <7>
+    type: "curator"  <8>
```

Comment:
The curator will be dropped, so I think we'd better not add the curator in the clusterlogging instance.